Makefile

```diff
@@ -532,7 +532,7 @@ artifacts: docker
 	@echo 'To be added'
 
 pilot/pkg/kube/config:
-	touch $@
+	@echo 'nothing needs to be done for' $@
```

Comment:
why don't you just remove this target? Thsi was part of my makefile cleanup PR but I didn't get to it to finish it fully.

Reply:
okay, removed.